src/main/java/dev/galacticraft/mod/mixin/EntityMixin.java

```diff
@@ -60,4 +69,42 @@ private void getTeleportTargetGC(ServerWorld destination, CallbackInfoReturnable
             cir.setReturnValue(new TeleportTarget(new Vec3d((double) pos.getX() + 0.5D, pos.getY(), (double) pos.getZ() + 0.5D), this.getVelocity(), this.yaw, this.pitch));
         }
     }
+
+    @Shadow
+    public abstract boolean updateMovementInFluid(Tag<Fluid> tag, double d);
+
+    @Shadow
+    public abstract boolean isOnFire();
+
+    @Shadow
+    private Vec3d pos;
+
+    @Shadow
+    protected UUID uuid;
+
+    @Shadow
+    public abstract boolean isPlayer();
+
+    @Shadow
+    private int id;
+
+    @Shadow
+    public abstract boolean isInvulnerable();
+
+    @Shadow
+    public abstract boolean damage(DamageSource source, float amount);
+
+    private static final DamageSource OIL_BOOM = (new DamageSource("gc.oilBoom")).setBypassesArmor();
```

Comment:
please move the `gc.oilBoom` to `Constant.java`

Reply:
Shoot, I was sure I did that. I may not have pushed that change.

Reply:
Looks like I put that in GalacticraftDamageSource. Is that okay?